pytensor/link/pytorch/dispatch/elemwise.py

```diff
@@ -11,9 +14,38 @@ def pytorch_funcify_Elemwise(op, node, **kwargs):
     scalar_op = op.scalar_op
     base_fn = pytorch_funcify(scalar_op, node=node, **kwargs)
 
-    def elemwise_fn(*inputs):
-        Elemwise._check_runtime_broadcast(node, inputs)
-        return base_fn(*inputs)
+    if isinstance(scalar_op, ScalarLoop):
```

Comment:
This is fine for now, in the long term, if we need many exceptions we can have a two level dispatch, where inside we dispatch to specific elemwise impl of scalar_ops which fall back to our back case below.

I would put this in a separate function for readability purposes though. so: `if isinstance(scalar_op, ScalarLoop): return elemwise_scalar_loop(op, node, **kwargs)`